my-dsl.md

```diff
@@ -2,37 +2,85 @@
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
 
+[Lout](http://www.adrianjwells.freeuk.com/lout.pdf) is a high-level markup language, similar to LaTex.
 
 # Domain
 _Describe the language's domain in five words._
 
+Typesetting documents, diagrams, and illustrations.
 
 # Computational model
 _What is the underlying computational model of this language? To answer this 
 question, provide a high-level description (no more than 100 words) of the 
 computation that occurs when someone executes a program in this language._
 
+Lout is a document formatter, so it reads a line of the Lout document and evaluates the expression(s) to generate a line of formatted text to add to the output document. It uses lazy evaluation to handle expressions, and can format computer program code in multiple languages, including C++, Java, Python, Perl, Ruby and Haskell.
 
```

Comment:
It would be interesting to know something about the layout engine (e..g, is it a box model, etc.). It would also be interesting to know how it uses lazy expressions. 

(although obviously the 100-word limit prevents you from expanding too much :))